Task 1/Minimum Vertex Cover/Question.md

```diff
@@ -0,0 +1,12 @@
+Problem
+=====================
+Link to the problem : https://www.geeksforgeeks.org/vertex-cover-problem-set-2-dynamic-programming-solution-tree/
```

Comment:
Add Markdown syntax for program link. Rest is fine.

Reply:
I have added the link in Markdown. Kindly review.